discretize/base/base_tensor_mesh.py

```diff
@@ -100,70 +118,56 @@ def __init__(self, h=None, x0=None, **kwargs):
         )
 
         # Ensure h contains 1D vectors
-        self.h = [utils.mkvc(x.astype(float)) for x in h]
+        self.h = [mkvc(x.astype(float)) for x in h]
 
     @property
-    def hx(self):
-        """Width of cells in the x direction"""
-        return self.h[0]
-
-    @property
-    def hy(self):
-        "Width of cells in the y direction"
-        return None if self.dim < 2 else self.h[1]
-
-    @property
-    def hz(self):
-        "Width of cells in the z direction"
-        return None if self.dim < 3 else self.h[2]
-
-    @property
-    def vectorNx(self):
+    def grid_nodes_x(self):
         """Nodal grid vector (1D) in the x direction."""
-        return np.r_[0., self.hx.cumsum()] + self.x0[0]
+        return np.r_[self.x0[0], self.h[0]].cumsum()
```

Comment:
General discussion point: is `x0` descriptive, or would something like `origin` be better?

Reply:
+1 for origin (though I think having `x0` as an alias would still be appropriate)